pyquil/tests/test_paulis.py

```diff
@@ -421,15 +421,21 @@ def test_exponentiate_identity():
     generator = PauliTerm("I", 1, 1.0)
     para_prog = exponential_map(generator)
     prog = para_prog(1)
-    result_prog = Program().inst([X(0), PHASE(-1.0, 0), X(0), PHASE(-1.0, 0)])
+    result_prog = Program()
     assert prog == result_prog
 
     generator = PauliTerm("I", 10, 0.08)
     para_prog = exponential_map(generator)
     prog = para_prog(1)
-    result_prog = Program().inst([X(0), PHASE(-0.08, 0), X(0), PHASE(-0.08, 0)])
+    result_prog = Program()
     assert prog == result_prog
 
+    pop = 2.1 * sI(0)
```

Comment:
A comment here explaining the rationale for this test might be helpful, since it's kind of an odd corner case. I'd personally be OK with dropping it altogether, but fine if you want to leave it to document the current behavior.